src/morse/api.clj

```diff
@@ -41,6 +41,28 @@
     (http/get url {:as :json :query-params query})))
 
 
+(defn get-file
+  "Gets url of the file"
+  [token file-id]
+  (let [url  (str base-url token "/getFile")
+        body ({:file_id file-id})
```

Comment:
Looks like something is missing here, isn't it?

Reply:
I have made mistake, the round brackets isn't needed after _body_.